src/Utils.js

```diff
@@ -157,11 +158,11 @@ export function coercePropValue(propValue) {
 
   // if propValue includes quotes, it should be
   // treated as a string
-  if (propValue.search(/"/) !== -1) {
-    return propValue.replace(/"/g, '');
+  if (/^(['"]).*\1$/.test(propValue.trim())) {
```

Comment:
producing a value once, and operating on that, rather than baking in the assumption that it will be idempotent when called twice, helps both clarity and performance :-p